README.md

```diff
@@ -69,6 +69,7 @@ $ install protobuf : git clone --progress -b v3.21.12 https://github.com/protoco
 ```
 #### Setup SDK local repository, build and test
 ```
+$ git clone https://github.com/eclipse-uprotocol/up-core-api.git
```

Comment:
Please change the git clone command to add `--branch uprotocol-core-api-1.5.5` so that you pull a specific version till we fix the build dependencies